doc/source/user_guide/deployment_juicefs_on_kubernetes.rst

```diff
@@ -0,0 +1,466 @@
+        Port:          6379/TCP
+        Host Port:     0/TCP
+        Command:
+          redis-server
+          /redis-master/redis.conf
+        State:          Running
+          Started:      Mon, 19 Jun 2023 09:58:20 +0000
+        Last State:     Terminated
+          Reason:       Error
+          Exit Code:    255
+          Started:      Mon, 19 Jun 2023 07:36:10 +0000
+          Finished:     Mon, 19 Jun 2023 09:58:01 +0000
+        Ready:          True
+        Restart Count:  5
+        Limits:
+          cpu:  100m
+        Requests:
+          cpu:  100m
+        Environment:
+          MASTER:  true
+        Mounts:
+          /redis-master from config (rw)
+          /redis-master-data from data (rw)
+          /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-s2tpw (ro)
+    Conditions:
+      Type              Status
+      Initialized       True
+      Ready             True
+      ContainersReady   True
+      PodScheduled      True
+    Volumes:
+      data:
+        Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
+        Medium:
+        SizeLimit:  <unset>
+      config:
+        Type:      ConfigMap (a volume populated by a ConfigMap)
+        Name:      example-redis-config
+        Optional:  false
+      kube-api-access-s2tpw:
+        Type:                    Projected (a volume that contains injected data from multiple sources)
+        TokenExpirationSeconds:  3607
+        ConfigMapName:           kube-root-ca.crt
+        ConfigMapOptional:       <nil>
+        DownwardAPI:             true
+    QoS Class:                   Burstable
+    Node-Selectors:              <none>
+    Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
+                                 node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
+    Events:                      <none>
+
+..
+
+
+Kubernetes
+----------
+Follow :ref:`kubernetes deployment document <deployment_kubernetes>` to initialize a K8s cluster on your machine.
+
+Install ``kubectl``, a command-line tool for interacting with Kubernetes clusters and verify its installation.
+
+.. code-block:: bash
+
+    $ kubectl version --client
+    WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
+    Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.4", GitCommit:"872a965c6c6526caa949f0c6ac028ef7aff3fb78", GitTreeState:"clean", BuildDate:"2022-11-09T13:36:36Z", GoVersion:"go1.19.3", Compiler:"gc", Platform:"linux/amd64"}
+    Kustomize Version: v4.5.7
+
+..
+
+
+JuiceFS Installation
+----------
+
+You can skip this installation part and jump directly to ``Deploy Cluster`` section.
+
+With the ``new_cluster`` function we provide in the next ``Deploy Cluster`` section, you can use JuiceFS directly without any installation.
```

Comment:
``new_cluster`` function just install the PV and PVC for xorbits namespace, right? So User must install juicefs csi on K8s.